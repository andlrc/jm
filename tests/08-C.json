{
	"@extends": ["08-B.json"],
	"@value": [
		{
			"@match": "[name=token]",
			"length": 11
		}
	]
}
