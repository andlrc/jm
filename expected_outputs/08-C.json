[
	{
		"name": "fieldA",
		"type": "varchar(64)"
	},
	{
		"name": "token",
		"type": "float",
		"length": 11
	}
]
